src/Viewer/renderers/point.jsx

```diff
@@ -336,8 +336,8 @@ export default React.memo(function Point(props) {
         shadowPointJXG.current.setAttribute({ face: newFace });
       }
       if (pointJXG.current.visProp.size !== SVs.selectedStyle.markerSize) {
-        pointJXG.current.visProp.size = SVs.selectedStyle.markerSize
-        shadowPointJXG.current.visProp.size = SVs.selectedStyle.markerSize
+        pointJXG.current.visProp.size = SVs.selectedStyle.markerSize;
```

Comment:
You need to use the transformed size
`normalizeSize(SVs.selectedStyle.markerSize, SVs.selectedStyle.markerStyle)`
rather than the original size
`SVs.selectedStyle.markerSize`
globally.

Reply:
I think I fixed this now, but please let me know.

Reply:
Look good.